### ISSUE TITLE
HLS VOD support? 

### ISSUE THREAD

**Issue by KudryavtsevDmitry** [ireader/media-server#205]
Hello. I have several questions about features

Does it support HLS VOD? If so, does it make it on-the-fly packaging (without writing/storing chunks on disk)?
Does it support mp4, mp3 containers?

**Comment by ireader** [comment]
> Does it support HLS VOD?  If so, does it make it on-the-fly packaging (without writing/storing chunks on disk)?
YES, libhls hls_media_handler callback with ts raw data, you can store on disk or in memory. 
see more at: libhls/demo/hls-segmenter-flv.cpp

> Does it support mp4, mp3 containers?
Yes, but don't support mp3.
libhls/hls-fmp4.h

**Comment by KudryavtsevDmitry** [comment]
Thanks for clear answer.

**Comment by ireader** [comment]
> Does it support HLS VOD?  If so, does it make it on-the-fly packaging (without writing/storing chunks on disk)?
YES, libhls hls_media_handler callback with ts raw data, you can store on disk or in memory. 
see more at: libhls/demo/hls-segmenter-flv.cpp

> Does it support mp4, mp3 containers?
Yes, but don't support mp3.
libhls/hls-fmp4.h
